app/build.gradle.kts

```diff
@@ -276,3 +276,6 @@ tasks.register("jacocoTestReport", JacocoReport::class) {
         include("outputs/code_coverage/debugAndroidTest/connected/*/coverage.ec")
     })
 }
+dependencies {
+    implementation("androidx.navigation:navigation-testing:2.7.7")
+}
```

Comment:
Not here the dependencies

Reply:
Yeah, automatically importing the dependencies does that for some reason, and I always move it. Thanks for catching it.